projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -198,7 +200,9 @@ export class QuickSearchComponent {
             }
             searchSection.result = searchResult;
             searchSection.isLoading = false;
-            this.selectFirst(true);
+            if (this.selectedItem == null) {
```

Comment:
We usually prefer `=== null` unless you are trying to specifically test for undefined also